src/legacy/core_plugins/metrics/public/np/plugin.ts

```diff
@@ -0,0 +1,41 @@
+/*
+ * Licensed to Elasticsearch B.V. under one or more contributor
+ * license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright
+ * ownership. Elasticsearch B.V. licenses this file to you under
+ * the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+// @ts-ignore
+import { tsvb } from '../tsvb_fn';
+// @ts-ignore
+import { MetricsVisProvider } from '../kbn_vis_types';
+import { TypesService } from '../../../visualizations/public/types/types_service';
+import { InterpreterService } from '../interpreter';
+
+export class Plugin {
+  private readonly visTypes: TypesService;
+  private readonly interpreter: InterpreterService;
+
+  constructor() {
+    this.visTypes = new TypesService();
```

Comment:
these shouldn't be here, but just passed in

Reply:
Removed